Makefile

```diff
@@ -26,6 +26,12 @@ OUTPUT ?= $(shell pwd)/_output
 INSTALL_PATH ?= $(OUTPUT)/bin
 LDFLAGS ?= -w -s -X k8s.io/component-base/version.gitVersion=$(VERSION)
 
+# flags for ecr-credential-provider artifact promotion
+UPLOAD ?= $(OUTPUT)/upload
+GCS_LOCATION ?= gs://test-ecr-build
```

Comment:
GCS location should be `gs://k8s-staging-provider-aws/releases/` before we land this PR, see [1]

[1] https://github.com/kubernetes/k8s.io/blob/main/artifacts/filestores/k8s-staging-provider-aws/filepromoter-manifest.yaml#L3

Reply:
The location should be set in the prow job, I think. The cloudbuild config below already forwards the env var.

Reply:
it's ok to set it in both places @olemarkus